extensions/azurecore/extension.webpack.config.js

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+//@ts-check
+
+'use strict';
+
+const withDefaults = require('../shared.webpack.config');
+
+module.exports = withDefaults({
+	context: __dirname,
+	entry: {
+		extension: './src/extension.ts'
+	},
+	externals: {
+		'ms-rest': 'commonjs ms-rest',
+		'request': 'commonjs request'
```

Comment:
Out of interest how did you decide which dependencies to put into their own 'require' files? Could we save on loading a lot of lodash stuff by referencing that as an external too?

Reply:
As far as I found lodash was only used by mssql for the decompress stuff for the service-downloader. How vscode's build scripts work you can only specify external dependency that you directly reference in the package.json. I put external dependencies as any that it, or one of it's dependencies, doesn't behave correctly for webpack to work.